src/angular-public/textarea/BUILD.bazel

```diff
@@ -47,7 +47,6 @@ ng_test_library(
         ":textarea",
         "//src/angular-core/testing",
         "//src/angular-public/form-field",
-        "@npm//@angular/cdk",
```

Comment:
Do you know why this was removed here?

Reply:
Yes, because I forgot to update the bazel files after removing the unused imports